### PR TITLE
Fix floating menu button on new recipe pages

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

Fixes floating menu button on new recipe pages, reported yesterday by @zeftilldeath. Seen here https://www.theguardian.com/books/little-library-cafe/2017/apr/06/novel-recipes-easter-fruitcake-from-84-charing-cross-road

## What is the value of this and can you measure success?

A proper navigation bar!

## Does this affect other platforms - Amp, Apps, etc?

No.

## Screenshots

** Before **

screen shot 2017-04-13 at 11 20 40

** After **

screen shot 2017-04-13 at 11 20 19

## Tested in CODE?

No.

### CODE REVIEW

File: common/app/model/content.scala [hunk 1]
```diff
@@ -83,7 +83,7 @@ final case class Content(
   lazy val isPaidContent: Boolean = tags.tags.exists{ tag => tag.id == "tone/advertisement-features" }
   lazy val campaigns: List[Campaign] = targeting.CampaignAgent.getCampaignsForTags(tags.tags.map(_.id))
   lazy val hasRecipeAtom: Boolean = atoms.fold(false)(a => a.recipes.nonEmpty)
-  lazy val showNewRecipeDesign: Boolean = hasRecipeAtom && ABNewRecipeDesign.switch.isSwitchedOn
+  lazy val showNewRecipeDesign: Boolean = hasRecipeAtom
```
Comment (NataliaLKB) [inline common/app/model/content.scala]: We still need && ABNewRecipeDesign.switch.isSwitchedOn here

File: common/app/mvt/MultiVariateTesting.scala [hunk 2]
```diff
@@ -27,6 +27,7 @@ object CommercialClientLoggingVariant extends TestDefinition(
   }
 }
 
+/** Watch out for "TODO: #new-recipe:" when removing the test */
```
Comment (NataliaLKB) [inline common/app/mvt/MultiVariateTesting.scala]: 💯

File: common/app/model/content.scala [hunk 3]
```diff
@@ -451,7 +451,8 @@ object Article {
       javascriptConfigOverrides = javascriptConfig,
       opengraphPropertiesOverrides = opengraphProperties,
       shouldHideHeaderAndTopAds = (content.tags.isTheMinuteArticle || (content.isImmersive && (content.elements.hasMainMedia || content.fields.main.nonEmpty))) && content.tags.isArticle,
-      contentWithSlimHeader = (content.isImmersive && content.tags.isArticle) || content.showNewRecipeDesign
+      contentWithSlimHeader = (content.isImmersive && content.tags.isArticle),
+      isNewRecipe = content.showNewRecipeDesign
```
Comment (NataliaLKB) [inline common/app/model/content.scala]: How do you feel about `isNewRecipeDesign` instead?

**Review by NataliaLKB — CHANGES_REQUESTED** [review]

**Comment by NataliaLKB** [comment]
Really great that you got to the bottom of this bug! :1st_place_medal:

File: common/app/model/content.scala (hunk 3 above)
Comment (gustavpursche) [inline common/app/model/content.scala]: I'm indifferent. I feel both shouldn't be there, but for consistency reasons you are absolutely right.

**Review by NataliaLKB — APPROVED** [review]

**Pull request merged**